contribs/gmf/src/directives/map.js

```diff
@@ -21,9 +22,8 @@ goog.require('ngeo.mapDirective');
  */
 gmf.mapDirective = function() {
   return {
-    scope: {},
-    bindToController: {
-      'map': '=gmfMapMap'
+    scope: {
+      'getMapFn': '&gmfMapMap'
     },
```

Comment:
> Just curious why you have eventually used scope instead of bindToController?

Because I did not want to pollute the directive's controller with a property (`getMapFn`) that I don't use in the directive's template.

> (by the way don't we need a bindToController: true line?)

I'd pollute the controller with `getMapFn` if I used `bindToController: true`.